examples/plot_linear_regression.py

```diff
@@ -0,0 +1,217 @@
+    n_rows, n_predictors = design_matrix.shape
+
+    if n_samples != n_rows:
+        raise ValueError('Number of rows in design matrix must be equal '
+                         'to number of observations')
+    if n_predictors != len(names):
+        raise ValueError('Number of regressor names must be equal to '
+                         'number of column in design matrix')
+
+    y = np.reshape(data, (n_samples, n_features))
+
+    # fit linear model with sklearn
+    # use fit intercept=false because b0 already in design matrix
+    linaer_model = LinearRegression(fit_intercept=False).fit(design_matrix, y)
+
+    # extract coefficients (i.e., betas)
+    betas = linaer_model.coef_.T
+
+    # save predictions and calculate rss
+    predictions = linaer_model.predict(design[predictors])
+    resid_sum_squares = sum((y - predictions) ** 2)  # sum of squared residuals
+
+    # dfs
+    df = n_rows - n_predictors
+
+    sqrt_noise_var = np.sqrt(resid_sum_squares / df).reshape(data.shape[1:])
+    design_invcov = linalg.inv(np.dot(design_matrix.T, design_matrix))
+    unscaled_stderrs = np.sqrt(np.diag(design_invcov))
+    tiny = np.finfo(np.float64).tiny
+    beta, stderr, t_val, p_val, mlog10_p_val = (dict() for _ in range(5))
+
+    for x, unscaled_stderr, predictor in zip(betas, unscaled_stderrs, names):
+        beta[predictor] = x.reshape(data.shape[1:])
+        stderr[predictor] = sqrt_noise_var * unscaled_stderr
+        p_val[predictor] = np.empty_like(stderr[predictor])
+        t_val[predictor] = np.empty_like(stderr[predictor])
+
+        stderr_pos = (stderr[predictor] > 0)
+        beta_pos = (beta[predictor] > 0)
+        t_val[predictor][stderr_pos] = (beta[predictor][stderr_pos] /
+                                        stderr[predictor][stderr_pos])
+        cdf = stats.t.cdf(np.abs(t_val[predictor][stderr_pos]), df)
+        p_val[predictor][stderr_pos] = np.clip((1. - cdf) * 2., tiny, 1.)
+        # degenerate cases
+        mask = (~stderr_pos & beta_pos)
+        t_val[predictor][mask] = np.inf * np.sign(beta[predictor][mask])
+        p_val[predictor][mask] = tiny
+        # could do NaN here, but hopefully this is safe enough
+        mask = (~stderr_pos & ~beta_pos)
+        t_val[predictor][mask] = 0
+        p_val[predictor][mask] = 1.
+        mlog10_p_val[predictor] = -np.log10(p_val[predictor])
+
+    return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+###############################################################################
+# list with subjects ids that should be imported
+subjects = list(range(2, 3))
+# create a dictionary containing participants data for easy slicing
+limo_epochs = {str(subject): limo.load_data(subject=subject) for subject in subjects}  # noqa
+
+# get key from subjects dict for easy slicing
+subjects = list(limo_epochs.keys())
+
+# drop EOGs and interpolate missing channels
+for subject in subjects:
+    limo_epochs[subject].drop_channels(['EXG1', 'EXG2', 'EXG3', 'EXG4'])
```

Comment:
Might want to set their type to EXG and pick data channels only instead.

Reply:
Cool thanks for the comment. I'm using picks now, although I still had to create a "exclude list" in order to create the info structure for the results evoked object.